.github/install_dependency_branch.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Define variables
+dependency_name=$1
+branch_name=$2
+
+root_dir=$(pwd)
+dependency_branch_name='main'
+branch_name_to_check=${branch_name}
+iterations=0
+
+while [ "$branch_name_to_check" != "$dependency_branch_name" ] && [ $iterations -lt 20 ]
+do
+  echo "Checking for ${dependency_name} branch: '${branch_name_to_check}'"
+  if
+    git ls-remote --exit-code \
+    --heads https://github.com/ihmeuw/"${dependency_name}".git "${branch_name_to_check}" == "0"
+  then
+    dependency_branch_name=${branch_name_to_check}
+    echo "Found matching branch: ${dependency_branch_name}"
+  else
+    echo "Could not find ${dependency_name} branch '${branch_name_to_check}'. Finding parent branch."
+    branch_name_to_check="$( \
```

Comment:
😵‍💫

Reply:
FYI none of this is new. It has been in VPH for months at this point